### PR TITLE
3 | Add webpack node babel server

### PR DESCRIPTION
**Pull request by cav2008**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "burritojs",
+  "version": "1.0.0",
+  "description": "web site scrapper",
+  "main": "index.js",
+  "directories": {
```
Comment (dario92) [inline package.json]: What is this property for ? 😄

File: package.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/dario92/burritojs/issues"
+  },
+  "homepage": "https://github.com/dario92/burritojs#readme",
+  "devDependencies": {
+    "babel-cli": "^6.18.0",
+    "babel-preset-es2015": "^6.18.0",
+    "babel-preset-stage-2": "^6.18.0",
+    "install": "^0.8.2",
```
Comment (dario92) [inline package.json]: Why we need this ?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    "npm": "^3.10.9",
+    "webpack": "^1.13.3"
```
Comment (dario92) [inline package.json]: Hey let's try to use `webpack 2` it has tree shaking.

**Review by dario92 — CHANGES_REQUESTED** [review]
Hey good job man! :) 
But you haven't plugged webpack 😄

File: package.json (hunk 1 above)
Comment (cav2008) [reply package.json]: @dario92 I dunno it was when you do npm init lol. Do you want me to delete this part?

File: package.json (hunk 2 above)
Comment (cav2008) [reply package.json]: @dario92 isn't this for the babel node lol? Or do you mean the install thing lol? if so I have no idea hahaha I will delete that one if you mean that

File: package.json (hunk 3 above)
Comment (cav2008) [reply package.json]: @dario92 ok cool I will install webpack 2

**Comment by cav2008** [comment]
@dario92 I added the changes. But I don't understand what you mean I haven't plugged webpack

**Pull request closed without merging**